fe/src/index.js

```diff
@@ -1,13 +1,21 @@
-import React from 'react';
-import ReactDOM from 'react-dom/client';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
-
-const root = ReactDOM.createRoot(document.getElementById('root'));
+import React from "react";
+import ReactDOM from "react-dom/client";
+import "./index.css";
+import UserProfile from "./pages/Profile/UserProfile";
+import reportWebVitals from "./reportWebVitals";
+import CreateReport from "./pages/Comments/Create/CommentCreate";
+import {BrowserRouter as Router, Routes, Route } from "react-router-dom";
+const root = ReactDOM.createRoot(document.getElementById("root"));
 root.render(
   <React.StrictMode>
-    <App />
+   
```

Comment:
- please format the code (apply to all)
https://www.youtube.com/watch?v=zd_aDbwr4pY